### ISSUE TITLE
integrate FASTPT into firecrown/CCL

### ISSUE THREAD

**Issue by beckermr** [LSSTDESC/firecrown#52]
We want to use FASTPT as a model for

1. extensions beyond the NLA IA models we use now
2. non-linear biasing

This issue encompasses the work needed to make that happen on the firecrown side. Related work on CCL is needed as well.

**Comment by slosar** [comment]
As of Feb 2020 the situation is as follows:
 * FASTPT has been implemented into CCL in Eulerian space
 * It is on the verge of being implemented into CCL in Lagrangias space
 * This should be a fairly easy task if my understanding of firecrown is kosher

Assigning to Blazek, who has too much time.

**Comment by jablazek** [comment]
I agree with this (except the part about having too much time). Right now, all that should be required is expanding the firecrown options to include the correct call to CCL. We had also discussed having FASTPT (and potentially other code) living outside of CCL and being passed in through generalized tracers/power spectra with the external plumbing in firecrown. I assume that we are OK sticking with the internal CCL option right now.

**Comment by slosar** [comment]
My take is that unless you have to, stick to CCL.

**Comment by marcpaterno** [comment]
This is closed by PR #174.
